public/index.html

```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width">
 
     <title>Snapshot Serengeti</title>
-    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700" />
-    <link href="http://cdn.leafletjs.com/leaflet-0.4.2/leaflet.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" />
+    <link href="https://cdn.leafletjs.com/leaflet-0.4.2/leaflet.css" rel="stylesheet" />
```

Comment:
That URL doesn't work.  I'll just drop it into the public directory for the moment.